frontend/src/app/layouts/admin-page/reports/report.component.html

```diff
@@ -0,0 +1,11 @@
+<div cdkDropList cdkDropListOrientation="horizontal" class="list" (cdkDropListDropped)="drop($event)">
```

Comment:
Pull request name is **Interceptor**, so I assume to see the interceptor code :) Please, clean this PR to include only changes that are relevant to interceptor